config/initializers/environment_check.rb

```diff
@@ -13,6 +13,16 @@ def mandatory_secret!(env, value)
         "mandatory in production because it's needed for the generation of secrets."
 end
 
+# Database Environment Variables have changed.
+if ENV["PORTUS_PRODUCTION_HOST"].present? && ENV["PORTUS_DB_HOST"].blank? ||
+    ENV["PORTUS_PRODUCTION_USERNAME"].present? && ENV["PORTUS_DB_USERNAME"].blank? ||
+    ENV["PORTUS_PRODUCTION_PASSWORD"].present? && ENV["PORTUS_DB_PASSWORD"].blank? ||
+    ENV["PORTUS_PRODUCTION_DATABASE"].present? && ENV["PORTUS_DB_DATABASE"].blank?
```

Comment:
This could be extended to any environment. As in, you can replace

```ruby
ENV["PORTUS_PRODUCTION_HOST"]
```

by:

```ruby
ENV["PORTUS_#{Rails.env.upcase}_HOST"]
```

And so on.

Reply:
according to af1b8b6 staging, development and test where previously hardcoded.